md.py

```diff
@@ -91,36 +91,41 @@ def _create_loop_disks(self, count, size):
 
         return ret
 
-    def setup(self, level=5, disks=None, ram_disks=None, loop_disks=None,
+    def setup(self, level=5, devs=None, ndisks=None, disk_type='ram',
```

Comment:
How about making the default for ndisks be 0?

Reply:
Should probably be 3

Reply:
Leaving as it was in the original, feature changes can be a separate pr if they are needed.